src/docker/Dockerfile

```diff
@@ -32,6 +32,7 @@ LABEL Description="This is the image for aether.jcabi.com" Vendor="jcabi.com" Ve
 WORKDIR /tmp
 
 ENV DEBIAN_FRONTEND=noninteractive
+ENV OPENJDK_VERSION 1.7.0_75
```

Comment:
@mkordas Should I simply say ``sudo apt-get install openjdk-7-jdk`` ? This what I find everywhere